jwst/superbias/bias_sub.py

```diff
@@ -79,4 +79,9 @@ def subtract_bias(input, bias):
     # of the science data
     output.data -= bias.data
 
+    if input.meta.exposure.zero_frame:
```

Comment:
Could add a simple comment here for this block, like "Subtract superbias from zeroframe data, if present"

Reply:
I added a comment.